idflakies-maven-plugin/pom.xml

```diff
@@ -89,5 +89,10 @@
             <artifactId>idflakies-core</artifactId>
             <version>2.0.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>edu.illinois</groupId>
+            <artifactId>starts-maven-plugin</artifactId>
+            <version>1.4-SNAPSHOT</version>
```

Comment:
Let's not depend on the SNAPSHOT version, does it work with the latest in central?